### DISCUSSION TITLE
Server-side authentication with next-intl

### DISCUSSION THREAD

**Discussion by EdVonTech** [supabase/supabase#16302] [Questions]
Guys, 

I am struggling to set up authentication in a server component in Nextjs with app router. 

I am following these docs that describe how to set up server-side authentication: https://supabase.com/docs/guides/auth/auth-helpers/nextjs#server-side.

I did the following steps
1. installed supabase, added env variables,  and adjusted middleware 
2. added app/auth/sign-up.ts
3. added app/auth/login.ts
4. app/auth/logout.ts
5. added a app/login.tsx component. 

However, the login is not possible. If I change everything to a client component and use event handlers, it is working. 

Does anyone has an example setup that uses server-side authentication? Would really appreciate ur help!

**Comment by jspm2013** [comment]
hi EdVonTech, did you come up with a solution? I am struggling with the same setup...

**Reply by s0h311** [reply]
Hi @jspm2013 are you getting any errors? If so please share the stack trace with us.
